### PR TITLE
Feature use css vars

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

File: src/components/general/Navbar/NavMenu/styles.scss [hunk 1]
```diff
@@ -58,7 +58,8 @@ $color-text-on-primary: get-text-color($color-primary);
     }
 
     &.active {
-        @include background-color($color-accent);
+        background-color: $color-accent;
+color: $color-text-on-accent;
```
Comment (tnagorra) [inline src/components/general/Navbar/NavMenu/styles.scss]: Fix indentation

File: src/stylesheets/_base.scss [hunk 2]
```diff
@@ -104,7 +104,8 @@ body {
             justify-content: center;
             height: $height-nagbar;
             font-weight: $font-weight-bold;
-            @include background-color($color-warning);
+            background-color: $color-warning;
+color: $color-text-on-warning
```
Comment (tnagorra) [inline src/stylesheets/_base.scss]: Fix indentation

File: src/views/CategoryEditor/DocumentPanel/DocumentNGram/styles.scss [hunk 3]
```diff
@@ -23,7 +23,8 @@
             font-family: inherit;
 
             &.active {
-                @include background-color($color-accent);
+                background-color: $color-accent;
+color: $color-text-on-accent;
```
Comment (tnagorra) [inline src/views/CategoryEditor/DocumentPanel/DocumentNGram/styles.scss]: Fix indentation

File: src/views/CategoryEditor/SubcategoryColumn/styles.scss [hunk 4]
```diff
@@ -67,7 +67,8 @@
             }
 
             &.active {
-                @include background-color($color-accent);
+                background-color: $color-accent;
+color: $color-text-on-accent;
```
Comment (tnagorra) [inline src/views/CategoryEditor/SubcategoryColumn/styles.scss]: Fix indentation

File: src/views/CategoryEditor/SubcategoryPropertyPanel/styles.scss [hunk 5]
```diff
@@ -23,7 +23,8 @@
             font-family: inherit;
 
             &.active {
-                @include background-color($color-accent);
+                background-color: $color-accent;
+color: $color-text-on-accent;
```
Comment (tnagorra) [inline src/views/CategoryEditor/SubcategoryPropertyPanel/styles.scss]: Fix indentation

File: src/views/Country/CountryDetail/CountryGeneral/styles.scss [hunk 6]
```diff
@@ -19,7 +19,8 @@ $region-detail-form-width: 320px;
         }
 
         .region-detail-form {
-            @include background-color($color-foreground);
+            background-color: $color-foreground;
+color: $color-text-on-foreground;
```
Comment (tnagorra) [inline src/views/Country/CountryDetail/CountryGeneral/styles.scss]: Fix indentation

File: src/views/LeadAdd/LeadActions/styles.scss [hunk 7]
```diff
@@ -25,12 +25,14 @@ $radius-button-border: 2px;
 
     .save-buttons {
         @extend %button-basic;
-        @include background-color($color-success);
+        background-color: $color-success;
+color: $color-text-on-success;
```
Comment (tnagorra) [inline src/views/LeadAdd/LeadActions/styles.scss]: Indentation fix is required.

File: src/views/LeadAdd/LeadActions/styles.scss [hunk 8]
```diff
@@ -25,12 +25,14 @@ $radius-button-border: 2px;
...
     }
 
     .remove-buttons {
         @extend %button-basic;
-        @include background-color($color-warning);
+        background-color: $color-warning;
+color: $color-text-on-warning;
```
Comment (tnagorra) [inline src/views/LeadAdd/LeadActions/styles.scss]: Indentation fix is required.

File: src/views/LeadAdd/LeadButtons/styles.scss [hunk 9]
```diff
@@ -1,7 +1,8 @@
 @import '~base-scss/base';
 
 .add-lead-buttons {
-    @include background-color($color-foreground);
+    background-color: $color-foreground;
+color: $color-text-on-foreground;
```
Comment (tnagorra) [inline src/views/LeadAdd/LeadButtons/styles.scss]: Indentation fix is required.

File: src/views/Login/styles.scss [hunk 10]
```diff
@@ -45,7 +45,8 @@ $height-deep-logo: 256px;
         width: $width-login-form;
 
         .hid-link-container {
-            @include background-color($color-foreground);
+            background-color: $color-foreground;
+color: $color-text-on-foreground;
```
Comment (tnagorra) [inline src/views/Login/styles.scss]: Indentation fix is required.

File: src/views/Login/styles.scss [hunk 11]
```diff
@@ -82,7 +83,8 @@ $height-deep-logo: 256px;
                 }
 
                 .or {
-                    @include background-color($color-foreground);
+                    background-color: $color-foreground;
+color: $color-text-on-foreground;
```
Comment (tnagorra) [inline src/views/Login/styles.scss]: Indentation fix is required.

File: src/views/Login/styles.scss [hunk 12]
```diff
@@ -95,7 +97,8 @@ $height-deep-logo: 256px;
         }
 
         .login-form {
-            @include background-color($color-foreground);
+            background-color: $color-foreground;
+color: $color-text-on-foreground;
```
Comment (tnagorra) [inline src/views/Login/styles.scss]: Indentation fix is required.

File: src/views/Login/styles.scss [hunk 13]
```diff
@@ -112,7 +115,8 @@ $height-deep-logo: 256px;
         }
 
         .register-link-container {
-            @include background-color($color-foreground);
+            background-color: $color-foreground;
+color: $color-text-on-foreground;
```
Comment (tnagorra) [inline src/views/Login/styles.scss]: Indentation fix is required.

File: src/views/StringManagement/Detail/InfoPane/styles.scss [hunk 14]
```diff
@@ -28,19 +28,22 @@ $width-error-pane: 320px;
 
         &.error {
             .title {
-                @include background-color($color-danger);
+                background-color: $color-danger;
+color: $color-text-on-danger;
```
Comment (tnagorra) [inline src/views/StringManagement/Detail/InfoPane/styles.scss]: Indentation fix is required.

File: src/views/StringManagement/Detail/InfoPane/styles.scss [hunk 15]
```diff
@@ -28,19 +28,22 @@ $width-error-pane: 320px;
...
             }
         }
 
         &.warning {
             .title {
-                @include background-color($color-warning);
+                background-color: $color-warning;
+color: $color-text-on-warning;
```
Comment (tnagorra) [inline src/views/StringManagement/Detail/InfoPane/styles.scss]: Indentation fix is required.

File: src/views/StringManagement/Detail/InfoPane/styles.scss [hunk 16]
```diff
@@ -28,19 +28,22 @@ $width-error-pane: 320px;
...
             }
         }
 
         &.info {
             .title {
-                @include background-color($color-info);
+                background-color: $color-info;
+color: $color-text-on-info;
```
Comment (tnagorra) [inline src/views/StringManagement/Detail/InfoPane/styles.scss]: Indentation fix is required.

File: src/views/StringManagement/Detail/styles.scss [hunk 17]
```diff
@@ -1,7 +1,8 @@
 @import '~base-scss/base';
 
 .header {
-    @include background-color($color-foreground);
+    background-color: $color-foreground;
+color: $color-text-on-foreground;
```
Comment (tnagorra) [inline src/views/StringManagement/Detail/styles.scss]: Indentation fix is required.

File: src/views/UserExports/styles.scss [hunk 18]
```diff
@@ -54,7 +54,8 @@
 
                         .row {
                             &.highlighted {
-                                @include background-color($color-accent);
+                                background-color: $color-accent;
+color: $color-text-on-accent;
```
Comment (tnagorra) [inline src/views/UserExports/styles.scss]: Indentation fix is required.

File: src/views/UserGroup/styles.scss [hunk 19]
```diff
@@ -85,7 +85,8 @@ $info-height: 144px;
 
         .usergroup-alt {
             @include shadow-medium;
-            @include background-color($color-foreground);
+            background-color: $color-foreground;
+color: $color-text-on-foreground;
```
Comment (tnagorra) [inline src/views/UserGroup/styles.scss]: Indentation fix is required.

File: src/views/UserProfile/UserEdit/styles.scss [hunk 20]
```diff
@@ -32,7 +32,8 @@
         :global {
             label {
                 @extend %button-like-link;
-                @include background-color($color-primary);
+                background-color: $color-primary;
+color: $color-text-on-primary;
```
Comment (tnagorra) [inline src/views/UserProfile/UserEdit/styles.scss]: Indentation fix is required.

File: src/widgetComponents/NumberMatrixInput/Row/ColumnElement/styles.scss [hunk 21]
```diff
@@ -1,6 +1,7 @@
 @import '~base-scss/base';
 
 .table-cell {
-    @include background-color($color-foreground);
+    background-color: $color-foreground;
+color: $color-text-on-foreground;
```
Comment (tnagorra) [inline src/widgetComponents/NumberMatrixInput/Row/ColumnElement/styles.scss]: Indentation fix is required.

File: src/widgetComponents/NumberMatrixInput/Row/styles.scss [hunk 22]
```diff
@@ -1,18 +1,22 @@
 @import '~base-scss/base';
 
 .table-header-row {
-    @include background-color($color-foreground);
+    background-color: $color-foreground;
+color: $color-text-on-foreground;
```
Comment (tnagorra) [inline src/widgetComponents/NumberMatrixInput/Row/styles.scss]: Indentation fix is required.

File: src/widgetComponents/NumberMatrixInput/Row/styles.scss [hunk 23]
```diff
@@ -1,18 +1,22 @@
...
     padding: $spacing-small;
 
     &.not-similar {
-        @include background-color($color-danger);
+        background-color: $color-danger;
+color: $color-text-on-danger;
```
Comment (tnagorra) [inline src/widgetComponents/NumberMatrixInput/Row/styles.scss]: Indentation fix is required.

File: src/widgetComponents/NumberMatrixInput/Row/styles.scss [hunk 24]
```diff
@@ -1,18 +1,22 @@
...
     }
 
     &.similar {
-        @include background-color($color-success);
+        background-color: $color-success;
+color: $color-text-on-success;
```
Comment (tnagorra) [inline src/widgetComponents/NumberMatrixInput/Row/styles.scss]: Indentation fix is required.

File: src/widgetComponents/NumberMatrixInput/Row/styles.scss [hunk 25]
```diff
@@ -1,18 +1,22 @@
...
     }
 
     &.partial-similar {
-        @include background-color($color-background);
+        background-color: $color-background;
+color: $color-text-on-background;
```
Comment (tnagorra) [inline src/widgetComponents/NumberMatrixInput/Row/styles.scss]: Indentation fix is required.

File: src/widgetComponents/NumberMatrixInput/styles.scss [hunk 26]
```diff
@@ -6,7 +6,8 @@
     overflow: auto;
 
     .table-header {
-        @include background-color($color-foreground);
+        background-color: $color-foreground;
+color: $color-text-on-foreground;
```
Comment (tnagorra) [inline src/widgetComponents/NumberMatrixInput/styles.scss]: Indentation fix is required.

File: src/components/general/Navbar/NavMenu/styles.scss (hunk 1 above)
Comment (virtualanup) [reply src/components/general/Navbar/NavMenu/styles.scss]: Yes

File: src/stylesheets/_base.scss (hunk 2 above)
Comment (virtualanup) [reply src/stylesheets/_base.scss]: No

**Pull request merged**